api/container_stop.go

```diff
@@ -10,15 +10,19 @@ import (
 // timeout value.  The timeout value the time before a forcible stop to the container is applied.
 // If the container cannot be found, a [ContainerNotFound](#ContainerNotFound)
 // error will be returned instead.
-func (c *API) ContainerStop(ctx context.Context, name string, timeout int) error {
+func (c *API) ContainerStop(ctx context.Context, name string, timeout int, ignoreStopped bool) error {
```

Comment:
Not necessary for now, but this seems like something we may eventually want to support a generic QueryOptions struct for as the list of options grows

Reply:
Yep, let's see if it grows again.